x/dualstaking/keeper/delegator_reward.go

```diff
@@ -85,6 +86,14 @@ func (k Keeper) GetAllDelegatorReward(ctx sdk.Context) (list []types.DelegatorRe
 func (k Keeper) CalcRewards(stakeEntry epochstoragetypes.StakeEntry, totalReward math.Int, delegations []types.Delegation) (providerReward math.Int, delegatorsReward math.Int) {
 	effectiveDelegations, effectiveStake := k.CalcEffectiveDelegationsAndStake(stakeEntry, delegations)
 
+	// Sanity check - effectiveStake > 0
+	if effectiveStake.IsZero() {
```

Comment:
effective stake can be zero in some cases. dont need the warning

Reply:
So just `return math.NewInt(0), math.NewInt(0)`?

Reply:
Like this? cfe747751c76af0a4a8e2e9f66ce55efd7d6bb46